snapshot/lib/snapshot/options.rb

```diff
@@ -182,16 +182,11 @@ def self.available_options
                                      description: "Separate the log files per device and per language",
                                      optional: true,
                                      is_string: false),
-        FastlaneCore::ConfigItem.new(key: :simultaneous,
-                                     env_name: "SNAPSHOT_EXECUTE_TESTS_SIMULTANEOUSLY",
-                                     description: "Take snapshots on multiple devices at once",
+        FastlaneCore::ConfigItem.new(key: :serialized_executions,
```

Comment:
how about `non_concurrent_simulators` I think `serialized` is a bit confusing.
Also this property is about disabling something, so I like how `non_concurrent_simulators` sounds like it is disabling a feature.